### DISCUSSION TITLE
Like always with this some kind of stuff – turn off JS please, for tests.

### DISCUSSION THREAD

**Discussion by guci0** [AntoineW/luge#48] [General]
As in the subject, it is clear that all this requires JS, but let's have something alternative, so that, for example, preloader, does not cause nothing to be displayed.

Btw great work! 🔥 & Keep watching...

**Comment by AntoineW** [comment]
Hi,
Thanks for your feedback. Have you tried using a noscript CSS file to achieve that? I assume that would be an easy workaround in that case.
